src/geode/mesh/builder/polygonal_surface_builder.cpp

```diff
@@ -388,12 +387,13 @@ namespace geode
                 PolygonEdge edge{ polygon, e };
                 const auto edge_id = polygonal_surface_.edge_from_vertices(
                     { vertices_id[e], vertices_id[e + 1] } );
-                polygon_edges[edge_id].emplace_back( std::move( edge ) );
+                polygon_edges[edge_id.value()].emplace_back(

```

Comment:
Should we add some ASSERT when using xxx.value()?

Reply:
If there is no value inside an optional, accessing its value throw.